app/assets/javascripts/student_profile/AcademicSummary.js

```diff
@@ -0,0 +1,548 @@
+              </td>
+              <td colSpan={12}>
+                {' '}
+              </td>
+              <td style={styles.core}>
+                85%
+              </td>
+              <td style={styles.strategic}>
+                {' '}
+              </td>
+              <td style={styles.intensive}>
+                73%
+              </td>
+              <td style={styles.core}>
+                92%
+              </td>
+              <td style={styles.strategic}>
+                {' '}
+              </td>
+              <td style={styles.intensive}>
+                84%
+              </td>
+              <td style={styles.core}>
+                93%
+              </td>
+              <td style={styles.strategic}>
+                {' '}
+              </td>
+              <td style={styles.intensive}>
+                84%
+              </td>
+              <td style={styles.core}>
+                95%
+              </td>
+              <td style={styles.strategic}>
+                {' '}
+              </td>
+              <td style={styles.intensive}>
+                91%
+              </td>
+              <td style={styles.core}>
+                97%
+              </td>
+              <td style={styles.strategic}>
+                {' '}
+              </td>
+              <td style={styles.intensive}>
+                93%
+              </td>
+              <td style={styles.core}>
+                96%
+              </td>
+              <td style={styles.strategic}>
+                {' '}
+              </td>
+              <td style={styles.intensive}>
+                91%
+              </td>
+              <td style={styles.core}>
+                97%
+              </td>
+              <td style={styles.strategic}>
+                {' '}
+              </td>
+              <td style={styles.intensive}>
+                93%
+              </td>
+              <td style={styles.core}>
+                98%
+              </td>
+              <td style={styles.strategic}>
+                {' '}
+              </td>
+              <td style={styles.intensive}>
+                95%
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    );
+  }
+
+  render() {
+    return (
+      <div className="AcademicSummary">
+        <div style={styles.textContainer}>
+          <span style={styles.caption}>
+            {this.props.caption + ':'}
+          </span>
+          <br />
+          <br />
+          <span style={styles.value}>
+            {(this.props.value === undefined) ? 'none' : this.props.value}
+          </span>
+          <HelpBubble
+            title="What do DIBELS levels mean?"
+            teaserText="(what is this?)"
+            content={this.getDibelsHelpContent()} />
+        </div>
+      </div>
+    );
+  }
+}
+
+SummaryWithoutSparkline.propTypes = {
+  caption: React.PropTypes.string.isRequired,
+  value: React.PropTypes.string // or null
+};
+
+export {
```

Comment:
Nice!

This is unrelated, but rather than exporting both, let's split these out into two separate files while we're at it.  I'm not sure why they were one file before, they seem pretty separate components.  Sometimes if there's a tiny "private" component that's good to define in the same file, but not export.  Or sometimes if you have two components that are really linked together you might define them in the same file and export both.  But these two seem unrelated, except for sharing a few minor styles like padding (which are fine to define locally in each file).